nbs/examples/ElectricityPeakForecasting.ipynb

```diff
@@ -0,0 +1,14449 @@
+{
```

Comment:
<p>I have found that users struggle with the <code>season_lenght</code>  parameter. Pointing to [this link](https://robjhyndman.com/hyndsight/seasonal-periods/) is particularly helpful. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Done</p>